src/transformers/modeling_bart.py

```diff
@@ -302,7 +303,7 @@ def __init__(self, config: BartConfig, embed_tokens):
         self.layers = nn.ModuleList([EncoderLayer(config) for _ in range(config.encoder_layers)])
         self.layernorm_embedding = LayerNorm(embed_dim) if config.normalize_embedding else nn.Identity()
         # mbart has one extra layer_norm
-        self.layer_norm = LayerNorm(config.d_model) if config.normalize_before else None
+        self.layer_norm = LayerNorm(config.d_model) if config.add_final_layer_norm else None
```

Comment:
no. This didn't break before because `add_final_layernorm` and `normalize_before` were always the same. (True for mbart, otherwise False). So this doesn't break mbart, and is False for all other models.